charts/tidb-operator/templates/controller-manager-rbac.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
 ---
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: v1
```

Comment:
It should be `rbac.authorization.k8s.io/v1`?

Reply:
`v1` is ok，i will add test result right now.